### PR TITLE
View/35 team mobile camilo

### PR DESCRIPTION
**Pull request by soycamis**

### CODE REVIEW

**Review by jresendiz27 — APPROVED** [review]

File: src/styl/sections/team.styl [hunk 1]
```diff
@@ -49,7 +49,8 @@
   flex-wrap wrap
   display flex
   justify-content flex-start
-  width 760px
+  //width 760px
```
Comment (ZhavaMan) [inline src/styl/sections/team.styl]: Remove comment

File: src/styl/sections/team.styl [hunk 2]
```diff
@@ -108,7 +109,8 @@
     height 20.9px
 
   & p
-    margin-top 56px
+    //margin-top 56px
```
Comment (ZhavaMan) [inline src/styl/sections/team.styl]: remove comment

File: src/styl/sections/team.styl [hunk 3]
```diff
@@ -172,15 +193,21 @@
...
     flex-direction column
     align-items center
 
   .team-photos
-    max-width 212px
+    padding-top 0
+    padding-left 0
+    overflow-y hidden
+    max-width 100%
+    //width 100%
```
Comment (ZhavaMan) [inline src/styl/sections/team.styl]: remove comment

**Review by ZhavaMan — APPROVED** [review]

**Review by ZhavaMan — CHANGES_REQUESTED** [review]
remove comments

**Review by ZhavaMan — APPROVED** [review]

**Pull request merged**